app/views/queue_items/index.html.haml

```diff
@@ -4,29 +4,31 @@
       %article
         %header
           %h2 My Queue
-        %table.table
-          %thead
-            %tr
-              %th(width="10%") List Order
-              %th(width="30%") Video Title
-              %th(width="10%") Play
-              %th(width="20%") Rating
-              %th(width="15%") Genre
-              %th(width="15%") Remove
-          %tbody
-            - @queue_items.each do |queue_item|
+        = bootstrap_form_tag url: "/update_queue" do 

```

Comment:
I would recommend using named routes instead of hardcoded url. Maintenance reasons and maintainable code should be your target.